test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -802,4 +805,28 @@ private static Throwable parallelRun(Throwable accumulate, ExecutorService runOn
         }
         return accumulate;
     }
+
+    @FunctionalInterface
+    private interface IgnoreThrowingRunnable extends ThrowingRunnable
+    {
+        void doRun() throws Throwable;
+
+        @Override
+        default void run()
+        {
+            try
+            {
+                doRun();
+            }
+            catch (Throwable e)
+            {
+                JVMStabilityInspector.inspectThrowable(e);
+            }
+        }
+
+        static IgnoreThrowingRunnable ignore(IgnoreThrowingRunnable fn)
```

Comment:
The static method only hints the compiler about the input type of `IgnoreThrowingRunnable`. 
I think it is easier to read to explicitly hint the type at the call-sites and remove this method. 

```java
// equivalent
//  IgnoreThrowingRunnable.ignore(MessagingService.instance()::shutdown)
(IgnoreThrowingRunnable) MessagingService.instance()::shutdown
```

Reply:
> The static method only hints the compiler about the input type of IgnoreThrowingRunnable.

Correct.

> I think it is easier to read to explicitly hint the type at the call-sites and remove this method.

Sure, will change.